source/templates/partials/_modal-test.twig

```diff
@@ -6,4 +6,23 @@
   btnReset: 'Annuler'
 } %}
 
-{% block modalContent %}This is modal content{% endblock %}
+{% block formContent %}
+
+<div class="form__wrapper">
+  <label class="form__label" for=""></label>
+  <input class="form__field" id="" type="text" name="" value="">
+</div>
```

Comment:
We also need the case where the input has an error (for example, if an input is required and user don't fill it, we wan tot display an error message).

- Duplicate this code, add a modifier class name on the wrapper for the error state
- To make an input be required, we could use the `required` attribute, but browser will show a message (https://media.geeksforgeeks.org/wp-content/uploads/20190503161910/reqt.png). We don't want this message, we want to display our own message, so we will use the `aria-required` attribute (https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute), and we will check if the input is required in development
- We need to "connect" input with this error message for screen readers, we can do it with aria-describedby attribute (https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute), try to implement it (read docs)

We also need the case where the input is disabled, can you do this case too?